webapp-user/src/models/Ajax.js

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright (C) 2017-2018 Canonical Ltd
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+var request =require('then-request');
+import axios from 'axios'
```

Comment:
I started with then-request and switched to axios... I should have remove the then-request dependency.

I was trying to get cookies to work and then-request seems to have a bug related to accessing cookies - since they never appear. I saw in another project that the Design team is using axios, so I thought it may be a better choice.